AsyncDisplayKit/ASVideoNode.h

```diff
@@ -47,6 +48,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, readwrite) BOOL shouldAutorepeat;
 
 @property (nonatomic, assign, readwrite) BOOL muted;
+@property (nonatomic, assign, readwrite) BOOL shouldAggressivelyRecoverFromStall;
```

Comment:
I think this should be named something else.  "Recover" is not clear.  What does this actually mean / do?  Play again before buffering is even finished?  Should it be a number that scales the aggressiveness in some way?

Please add a comment describing what it does, consider if it should be a float / etc, and think about a more technically descriptive name.  @Xinchi this is important to change in the next few days to minimize folks adopting API that we change.

Overall, this PR looks like _really great work!_  Thanks so much for your contributions, @Xinchi :)